### ISSUE TITLE
Добавить историю нарушений политики в отдельном всплывающем окне

### ISSUE THREAD

**Issue by Maksim2355** [kirich1409/StrictModeCompat#5]
Похожий функционал реализован в более старой либе

https://github.com/nshmura/strictmode-notifier/blob/master/assets/video.gif

Возможно ли такая фича в текущей?

**Comment by kirich1409** [comment]
Да, ничего не мешает сделать доп activity, которое будет собирать всю информацию об StrictMode.
